shared/api/image.go

```diff
@@ -102,3 +103,16 @@ type ImageAliasesEntry struct {
 
 	Name string `json:"name" yaml:"name"`
 }
+
+// GetExistingAliases returns the intersection between a list of aliases and all the existing ones.
+func GetExistingAliases(aliases []string, allAliases []ImageAliasesEntry) []ImageAliasesEntry {
```

Comment:
This should also be in lxc/utils.go at least for now. "shared/api" is really meant to just be structs that match the API messages.